src/bridge/mod.rs

```diff
@@ -35,7 +35,11 @@ fn set_windows_creation_flags(cmd: &mut Command) {
 fn platform_build_nvim_cmd(bin: &str) -> Option<Command> {
     if SETTINGS.get::<CmdLineSettings>().wsl {
         let mut cmd = Command::new("wsl");
-        cmd.arg(bin);
+        cmd.args(&[
```

Comment:
What is this change for?

Reply:
I think this was a merge error

Reply:
It's quite curious. That command looks like it would set PATH to an empty string. Was curious if wsl had some special properties.

Reply:
Aaa nvm, it prints error to null.